frappe/utils/safe_exec.py

```diff
@@ -152,7 +152,15 @@ def get_safe_globals():
 			avg = frappe.db.avg,
 			count = frappe.db.count,
 			min = frappe.db.min,
-			max = frappe.db.max
+			max = frappe.db.max,
+			commit = frappe.db.commit
+		)
+
+		out.frappe.cache = NamespaceDict(
+			get_value = frappe.cache().get_value,
+			set_value = frappe.cache().set_value,
+			hset = frappe.cache().hset,
+			hget = frappe.cache().hget
```

Comment:
Can we have consistency in the APIs? Have `frappe.cache().set_value` instead of `frappe.cache.set_value`...so on and so forth?